README.md

```diff
@@ -13,41 +13,28 @@ _Create workflows that enable you to use Continuous Integration (CI) for your pr
 
 </header>
 
-<!--
-  <<< Author notes: Step 4 >>>
-  Start this step by acknowledging the previous step.
-  Define terms and link to docs.github.com.
--->
+## Step 5: Merge your pull request
 
-## Step 4: Add branch protections
+_Almost there! :heart:_
 
-_Great job uploading test reports! :partying_face:_
+You can now [merge](https://docs.github.com/get-started/quickstart/github-glossary#merge) your pull request!
 
-Take a look at the merge box, you'll notice you can merge this even though the review process hasn't been met.
+### :keyboard: Activity: Merge your pull request
 
-Protected branches ensure that collaborators on your repository cannot make irrevocable changes to branches. Enabling protected branches also allows you to enable other optional checks and requirements, like required status checks and required reviews.
+1. Go to the **Pull requests** tab.
+1. Resolve any merge conflicts. <!-- TODO: figure out a better way to deal with the merge conflicts. -->
+1. Click **Merge pull request**.
```

Comment:
I didn't even understand what the problem was. Of course using less dependencies and fancy play we do not have to deal with these.